storage/innobase/xtrabackup/src/xbcloud/xbcloud.cc

```diff
@@ -317,6 +339,31 @@ static struct my_option my_long_options[] = {
      &opt_s3_api_version, &opt_s3_api_version, &s3_api_version_typelib,
      GET_ENUM, REQUIRED_ARG, 0, 0, 0, 0, 0, 0},
 
+    {"azure-storage-account", OPT_AZURE_ACCOUNT, "AZURE storage account ",
+     &opt_azure_account, &opt_azure_account, 0, GET_STR_ALLOC, REQUIRED_ARG, 0,
+     0, 0, 0, 0, 0},
+
+    {"azure-container-name", OPT_AZURE_CONTAINER, "AZURE container name ",
+     &opt_azure_container, &opt_azure_container, 0, GET_STR_ALLOC, REQUIRED_ARG,
+     0, 0, 0, 0, 0, 0},
+
+    {"azure-access-key", OPT_AZURE_ACCESS_KEY, "AZURE access key.",
+     &opt_azure_access_key, &opt_azure_access_key, 0, GET_STR_ALLOC,
+     REQUIRED_ARG, 0, 0, 0, 0, 0, 0},
+
+    {"azure-development-storage", OPT_AZURE_DEVELOPMENT_STORAGE,
+     "use azure development storage", &opt_azure_development_storage,
```

Comment:
I would mention the name of the sandbox, azurite IIRC

Reply:
modified the help message.